changelog.d/11243.feature

```diff
@@ -0,0 +1 @@
+Allow events to be created without `prev_events` (only `auth_events`).
```

Comment:
```suggestion
Allow specific, experimental events to be created without `prev_events`. Used by MSCXXXXX
```

Or something? And its probably a misc? Or maybe we actually just want to have a generic `Improve support for MSCXX` changelog so all these PRs go into the same changelog entry. End users/server admins won't care about the guts of implementations like this (which is the audience of these changelogs)